.circleci/config.yml

```diff
@@ -1,8 +1,8 @@
 version: 2.1
 
 orbs:
+  hmpps: ministryofjustice/hmpps@2.2
```

Comment:
We need the new plugin for helm, but the new plugin introduced ktlint too so lots of what feel like unimportant linting changes ahead!